Add a --warnings option

jmgrady opened the pull request:
When the --warnings (or -w) option is specified, the script will print a
warning on stderr when an error is detected instead of raising an
exception.

jmgrady committed:
Add a --warnings option

When the --warnings (or -w) option is specified, the script will print a
warning on stderr when an error is detected instead of raising an
exception.

imnasnainaec commented:
@jmgrady May want to add `--warnings` to the README. Otherwise, this looks mergable.

jmgrady committed:
Update help text on main.py and README.md

jmgrady committed:
Python reformatting

jmgrady commented:
> @jmgrady May want to add `--warnings` to the README. Otherwise, this looks mergable.

Added `help` parameters to the Option definitions and added instructions in the README to run `python main.py translate --help` to see available options.
